client/state/data-layer/wpcom/read/feed/index.js

```diff
@@ -32,35 +47,31 @@ export function initiateFeedSearch( store, action ) {
 				exclude_followed: action.payload.excludeFollowed,
 				sort: action.payload.sort,
 			},
-			onSuccess: action,
-			onFailure: action,
-		} )
+		},
+		action
 	);
 }
 
-export function receiveFeeds( store, action, apiResponse ) {
-	const feeds = map( apiResponse.feeds, feed => ( {
-		...feed,
-		feed_URL: feed.subscribe_URL,
-	} ) );
-
-	const total = apiResponse.total > 200 ? 200 : apiResponse.total;
-	store.dispatch( receiveFeedSearch( queryKey( action.payload ), feeds, total ) );
+export function receiveReadFeedSearchSuccess( action, data ) {
+	const { feeds, total } = data;
+	return bypassDataLayer( receiveFeedSearch( queryKey( action.payload ), feeds, total ) );
```

Comment:
I don't think we need to use `bypassDataLayer` here since the data-layer isn't listening for this action.